gammapy/analysis/config.py

```diff
@@ -218,7 +218,14 @@ class ObservationsConfig(GammapyBaseConfig):
     obs_file: Path = None
     obs_cone: SpatialCircleConfig = SpatialCircleConfig()
     obs_time: TimeRangeConfig = TimeRangeConfig()
-    required_irf: List[RequiredHDUEnum] = ["aeff", "edisp", "psf", "bkg"]
+    required_hdus: List[RequiredHDUEnum] = [
```

Comment:
A priori this is a different attribute in `ObservationConfig` so I would say that https://docs.gammapy.org/dev/development/dev_howto.html#removing-an-attribute
But I don't know whether this will work in this specific case.